lethal-squirrels/Bangazon-Site
# Ben product pic

## REFERENCE

Ticket: #14 

## DESCRIPTION
Added a field to "CreateProduct" view to upload a file. Takes any file! Don't abuse it.
Saves the file to a local directory. Saves the relative path to the product.ImgPath
Added an <img> to product detail where it displays the img saved.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In the package manager, run `update-database`. 
3. Test functionality
	* Add a product
	* Add a pic.
	* Check that the pic is saved and shown on the details page.
	* Maybe click around and make sure I didn't break anything
